server/db/index.js

```diff
@@ -58,6 +58,21 @@ const Favorite = sql.define('favorites', {
     primaryKey: true,
     autoIncrement: true,
   },
+  userId: {
```

Comment:
userId and recipeId already exist on the Favorites table because of the relationship definitions at the bottom. If you open the psql shell and describe each table (the command is "\d favorites") it will show you the foreign keys associated with the table. So lines 61 - 72 can be removed